Documentation/AdvancedUsage.md

```diff
@@ -522,13 +532,14 @@ let manager = ServerTrustManager(evaluators: serverTrustPolicies)
 ```
 
 This `ServerTrustManager` will have the following behaviors:
+
 - `cert.example.com` will always use certificate pinning with default and host validation enabled , thus requiring the following criteria to be met in order to allow the TLS handshake to succeed:
 	- Certificate chain *must* be valid.
 	- Certificate chain *must* include one of the pinned certificates.
 	- Challenge host *must* match the host in the certificate chain's leaf certificate.
 - `keys.example.com` will always use public key pinning with default and host validation enabled, thus requiring the following criteria to be met in order to allow the TLS handshake to succeed:
 	- Certificate chain *must* be valid.
-	- Leaf certificates *must* include one of the pinned public keys.
+	- Certificate chain *must* include one of the pinned public keys.
```

Comment:
It evaluates the entire chain for both certs and public keys looking for a match.

Reply:
👍